app/services/email_service.rb

```diff
@@ -87,11 +87,20 @@ def self.email_address_allowed?(address, allowed_emails)
     allowed = false
     allowed_array = allowed_emails.split(",")
     allowed_array.each do |allowed_domain_or_address|
-      allowed_domain_or_address.strip!
-      allowed_domain_or_address.gsub!('.', '\.') #change . to be \. to only match a dot, not any char
-      if address =~ /#{allowed_domain_or_address}$/i
-        allowed = true
-        break
+      if allowed_domain_or_address =~ /^\/.+/
```

Comment:
You don't need the `.+` here.

Reply:
good point, there's no $ in the end
Maybe I won't change now anyway as ready and tested with this, but good comment to improve my RegExps in the future 👍

Reply:
or actually I did fix that too, as some updates (WTI) also still coming in..